From: Russ Allbery <rra@debian.org>
Date: Sun, 14 Aug 2016 14:14:47 -0700
Subject: Fix library install path

The upstream top-level configure script doesn't correctly handle
--program-suffix without this tweak.
---
 configure.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index e783564..c938531 100644
--- a/configure.in
+++ b/configure.in
@@ -599,13 +599,13 @@ fi
 dnl The "right" way of using program_transform_name, program_prefix, and
 dnl program_suffix expects you to use AC_ARG_PROGRAM here and do transformation
 dnl in the Makefile, but we want the values now, so we do it ourselves.
-EXENAME="tf"
+EXENAME="tf" PROGNAME="tf"
 test "${program_prefix}" != "NONE" && EXENAME="${program_prefix}${EXENAME}"
 test "${program_suffix}" != "NONE" && EXENAME="${EXENAME}${program_suffix}"
 if test "${program_transform_name}" != "NONE"; then
     EXENAME=`echo "${EXENAME}" | sed "${program_transform_name}"`
 fi
-LIBNAME="${EXENAME}-lib"
+LIBNAME="${PROGNAME}-lib"
 
 if test "$enable_symlink" = "no"; then
     SYMLINK=""
@@ -616,6 +616,7 @@ else
 fi
 
 AC_SUBST(EXENAME)
+AC_SUBST(PROGNAME)
 AC_SUBST(LIBNAME)
 AC_SUBST(SYMLINK)
 
